### DISCUSSION TITLE
'place-items-center' doesn't work on y-axis

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#6120] [Help]
When I put `place-items-center` class in the body tag it doesn't center the content on the y-axis, but in this [video](https://youtu.be/WJDw1J7FZnE) it works just fine, here is my code:
https://play.tailwindcss.com/ry8FtwtzSO

**Comment by MartinHughes-BPC** [comment]
It's because you're specifying a body tag, and the page already has one. Try changing it to a div instead.
